architecture/additional_concepts/authorization.adoc

```diff
@@ -196,58 +197,123 @@ JSON file. This allows the project *admin* to bind users to roles that are
 defined only in the `_<projectname>_` local policy.
 
 == Security Context Constraints
+In addition to link:#overview[authorization policies] that control what a user
+can do, OpenShift provides _security context constraints_ (SCC) that control the
+actions that a link:../core_objects/kubernetes_model.html#pod[pod] can perform
+and what it has the ability to access.
+
+SCCs are objects that define a set of conditions that a pod must run with in
+order to be accepted into the system. They allow an administrator to control the
+following:
+
+. Running of
+link:../../admin_guide/install/setup.html#security-warning[privileged
+containers].
+. Capabilities a container can request to be added.
+. Use of host directories as volumes.
+. The SELinux context of the container.
+. The user ID.
+
+Two SCCs are added to the cluster by default, _privileged_ and _restricted_,
+which are viewable using the CLI:
```

Comment:
Meant to double-check whether this was a cluster-admin only ability. Now I see that it seems to be:

```
Error from server: User "alice" cannot list all securitycontextconstraints in the cluster
```

Will update.